modules/@apostrophecms/attachment/index.js

```diff
@@ -212,22 +155,9 @@ module.exports = {
   },
   handlers(self) {
     return {
-      'apostrophe:modulesReady': {
-        // Delay setting up the collection until other modules
-        // are ready in a normal startup. Necessary because uploadfs must
-        // be available during asset builds but the database must not be
-        async enableCollection() {
-          self.db = await self.apos.db.collection('aposAttachments');
-          await self.db.createIndex({ docIds: 1 });
-          await self.db.createIndex({ trashDocIds: 1 });
-        },
-        addFixLengthPropertyMigration() {
-          self.addFixLengthPropertyMigration();
-        }
-      },
       'apostrophe:destroy': {
         async destroyUploadfs() {
-          await Promise.promisify(self.uploadfs.destroy)();
+          await Promise.promisify(self.apos.uploadfs.destroy)();
```

Comment:
Should this handler go in `@apostrophecms/uploader`?

Reply:
It should move yes, good catch.